app/Http/Controllers/Api/TechnicienSyncController.php

```
@@ -141,7 +142,40 @@ public function sync(Request $request)
+                        $failed++;
+
+                        continue;
+                    }
+
+                    if ($latitude < -90 || $latitude > 90 || $longitude < -180 || $longitude > 180) {
+                        $failed++;
+
+                        continue;
+                    }
+
+                    InterventionGpsTrack::create([
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\Interventions\InterventionGpsTrack::create().


(staticMethod.notFound)